docs/reference/mount.md

```diff
@@ -167,9 +167,9 @@ You can use lakeFS’s existing [Role-Based Access Control mechanism](../referen
 
 ### Does data pass through the lakeFS server when using lakeFS Mount?
 
-lakeFS Mount leverages  pre-signed URLs to read data directly from the underlying object store, meaning data doesn’t  pass through the lakeFS server. By default, presign is disabled. To enable it, use:
+lakeFS Mount leverages  pre-signed URLs to read data directly from the underlying object store, meaning data doesn’t  pass through the lakeFS server. By default, presign is enabled. To disable it, use:
```

Comment:
didnt we agree that presigned is only enabled by default if supported by the underlying object store or false otherwise?

Reply:
@ozkatz The new behavior introduced in [PR](https://github.com/treeverse/fluffy/pull/290) is the same as `lakectl fs upload` exactly. 

`everest` will behave the following way:

- if `presign` is enabled by the server it **will use** `presign` otherwise if not support by the server it will not use `presign` 
- if user set explicitly `--presign=false` we will **not use** `presign`.